pkg/controller/hibernation/hibernation_controller.go

```diff
@@ -577,6 +743,39 @@ func (r *hibernationReconciler) nodesReady(cd *hivev1.ClusterDeployment, remoteC
 	return true, nil
 }
 
+func (r *hibernationReconciler) operatorsReady(remoteClient client.Client, logger log.FieldLogger) (bool, error) {
+	logger.Debug("Checking if ClusterOperators are ready")
+	coList := &configv1.ClusterOperatorList{}
+	err := remoteClient.List(context.TODO(), coList)
```

Comment:
I am confused how it works without mentioning the namespace of cd - maybe that was the original intent but missed?

Reply:
The `remoteClient` arg points to the *remote* cluster. This func is checking on the health of all the ClusterOperators there.